src/bootstrap/native.rs

```diff
@@ -175,6 +175,14 @@ impl Step for Llvm {
             cfg.define("LLVM_ENABLE_ZLIB", "OFF");
         }
 
+        // Are we compiling for iOS/tvOS?
+        if target.contains("apple") && !target.contains("darwin") {
+            cfg.define("CMAKE_OSX_SYSROOT", "/");
+            cfg.define("CMAKE_OSX_DEPLOYMENT_TARGET", "");
```

Comment:
These seem a bit out there -- can we include a link to some documentation or explain here why `/` and empty string are the right settings?

Reply:
If these aren't set, it'll try to use a MacOSX sysroot by default, leading to a compiler error.

I forgot where I found this, it was some StackOverflow question I believe.

Reply:
I think adding a comment in the code that says something like:

// If these magic values are not set, it will try to make a Mac OS X sysroot by default, leading to a compiler error.

would be great.

Reply:
Resolved by https://github.com/rust-lang/rust/pull/73903/commits/4fd1c77a24288383600b02a7e3f11bf5eb718a32